mypyc/ir/ops.py

```diff
@@ -1138,6 +1138,33 @@ def accept(self, visitor: 'OpVisitor[T]') -> T:
         return visitor.visit_raise_standard_error(self)
 
 
+class CFunctionCall(RegisterOp):
```

Comment:
Bikeshedding: for consistency with the existing `Call` primitive, a name such as `CallC` would arguably be more consistent. Later on we may want to merge `Call` and `CallC`.